Misc/NEWS.d/next/Library/2019-05-24-18-16-07.bpo-37035.HFbJVT.rst

```diff
@@ -0,0 +1,2 @@
+Don't log OSError based exceptions if a fatal error has occurred in asyncio
```

Comment:
Please expand this to include the motivation, e.g. that the errors will still propagate to the user code, it's just logging them is a bit pointless.